src/state/slices/common-selectors.ts

```diff
@@ -64,7 +64,8 @@ export const selectPortfolioCryptoPrecisionBalanceByFilter = createCachedSelecto
   selectAssetIdParamFromFilter,
   (assets, accountBalances, assetBalances, accountId, assetId): string => {
     if (!assetId) return '0'
-    const precision = assets?.[assetId]?.precision ?? 0
+    const precision = assets?.[assetId]?.precision
+    if (!precision) return '0'
```

Comment:
We might want to add a comment here re: why we want to early return here, to avoid megabillons when missing precision (caused by some locals?).